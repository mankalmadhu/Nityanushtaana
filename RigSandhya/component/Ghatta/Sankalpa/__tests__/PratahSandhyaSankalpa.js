import React from 'react';
import ಪ್ರಾತಃಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ from '../PratahSandhyaSankalpa';
import Renderer from 'react-test-renderer';

jest.mock('../../Gayatri/GayatriMantra', () => 'ಗಾಯತ್ರೀಮ೦ತ್ರ' );
jest.mock('../../Gayatri/Vyahruti', () => 'ವ್ಯಾಹೃತಿ' );
jest.mock('../SankalpaPurvanga', () => 'ಸ೦ಕಲ್ಪಪೂರ್ವಾ೦ಗ' );
jest.mock('../SankalpaShershike', () => 'ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ' );
jest.mock('../Oomapo', () => 'ಓಮಾಪೋ' );

test("ಪ್ರಾತಃಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ", () => {
  const pratahSandhyaSankalpa = Renderer.create(
    <ಪ್ರಾತಃಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ />
  ).toJSON();

  expect(pratahSandhyaSankalpa).toMatchSnapshot();

});