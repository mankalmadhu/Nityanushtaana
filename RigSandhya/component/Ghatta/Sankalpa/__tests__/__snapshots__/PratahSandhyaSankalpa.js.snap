// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರಾತಃಸ೦ಧ್ಯಾಸ೦ಕಲ್ಪ 1`] = `
<View>
  <ಸ೦ಕಲ್ಪಶೀರ್ಷಿಕೆ />
  <ಸ೦ಕಲ್ಪಪೂರ್ವಾ೦ಗ />
  <ವ್ಯಾಹೃತಿ />
  <ಗಾಯತ್ರೀಮ೦ತ್ರ />
  <ಓಮಾಪೋ />
  <ವಿಷಯ
    ಮಾಹಿತಿ="ಮಮ ಉಪಾತ್ತ ಸಮಸ್ತ ದುರಿತಕ್ಷಯದ್ವಾರಾ ಶ್ರೀ ಪರಮೇಶ್ವರ ಪ್ರೀತ್ಯರ್ಥ೦ ಪ್ರಾತಃ ಸ೦ಧ್ಯಾ೦ ಉಪಾಸ್ಯೇ ॥"
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  />
</View>
`;
